### DISCUSSION TITLE
How do I uninstall this tool completely?

### DISCUSSION THREAD

**Discussion by mustafa-ans** [pear-devs/pear-desktop#1230] [Q&A]
Is there an uninstaller that I need to use to remove this application from my system? I can't see any folder in the windows C drive. I will really appreciate your help. Thank you

**Comment by inson1** [comment]
This isnt enough? (Uninstall option)
![ImageGlass_KY4hX58hkv](https://github.com/th-ch/youtube-music/assets/75314629/52ee7178-dd1d-4f53-b1cb-b658c1746787)

**Reply by mustafa-ans** [reply]
I already tried that but I saw a file named yt in my c drive so I thought it wasn't removed completely? Maybe it's a backup of the settings I had on the app idk

**Reply by mustafa-ans** [reply]
Thank you for answering btw @inson1

**Reply by ArjixWasTaken** [reply]
we don't create/use files named `yt`

**Comment by markpk69** [comment]
IObit Uninstaller works good

**Reply by mustafa-ans** [reply]
Will give it a shot, thank you for taking the time to answer :)
